src/styles/blocks/card-style.scss

```diff
@@ -0,0 +1,137 @@
+
+
+* {
+  margin: 0;
```

Comment:
It's not a good practice to reset margins using universal selector
![image](https://github.com/mate-academy/layout_product-cards/assets/96409563/bc75cfa2-a3ee-410d-9381-d5cbd3fdb318)